topdown/cache/cache_test.go

```diff
@@ -139,6 +140,71 @@ func TestInsert(t *testing.T) {
 	if !found {
 		t.Fatal("Expected key \"foo5\" in cache")
 	}
+
+	// replacing an existing key should not affect cache size
+	cache = NewInterQueryCache(config)
+
+	cacheValue6 := newInterQueryCacheValue(ast.StringTerm("bar6").Value, 10)
```

Comment:
Hmm funny. I can see that it was used like that above, too, but there's really no need for a Term here:
```suggestion
	cacheValue6 := newInterQueryCacheValue(ast.String("bar6"), 10)
```
this should do the trick, too.

Reply:
Should I change this for the other tests as well?